### PR TITLE
bettercap: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by dawidd6**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by dawidd6** [comment]
Depends on: `https://www.netfilter.org/projects/libnetfilter_queue`

Which is not in Homebrew yet. Should make it a linux only formula or contribute it to upstream? @iMichka @issyl0

File: Formula/bettercap.rb [hunk 1]
```diff
@@ -14,9 +14,11 @@ class Bettercap < Formula
   depends_on "dep" => :build
   depends_on "go" => :build
   depends_on "pkg-config" => :build
+  depends_on "libpcap" if OS.linux?
```
Comment (iMichka) [inline Formula/bettercap.rb]: We prefer `unless OS.mac?` here. It's more for historical reason, and I would like to keep this consistent everywhere.

Comment (iMichka) [reply Formula/bettercap.rb]: Also, linux dependencies should be added at the end of the list. This allows us to separate them logically from the other dependencies. And it is often helpful to have them at the end when doing merges, when the mac dependencies are changed, there is a better chance to not have a merge conflict.

**Comment by iMichka** [comment]
I think we can add these to linux only. Not sure these even build/work on mac.

**Comment by dawidd6** [comment]
```
Error: 2 problems in 1 formula detected
bettercap:
  * C: 18: col 3: FormulaAudit/DependencyOrder: dependency "libpcap" (line 18) should be put before dependency "libusb" (line 17)
  * C: 19: col 3: FormulaAudit/DependencyOrder: dependency "libnetfilter-queue" (line 19) should be put before dependency "libusb" (line 17)
==> FAILED
```

I should probably ignore this, cause we want linux specific deps to be at the bottom right @iMichka?

**Pull request closed without merging**